### PR TITLE
[MIG] component, component_event, test_component: Migration to 13.0

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,18 +32,32 @@ jobs:
...
+  - pip install sphinx sphinx_bootstrap_theme sphinx-intl
+  - pip install git+https://github.com/OCA/odoo-sphinx-autodoc.git
+
+services:
+  - postgresql
+
 
 script:
-  - travis_run_tests
+  - if [ "$DOCS" != "1" ] ; then travis_run_tests; fi
```
Comment (guewen) [inline .travis.yml]: You shouldn't need this test, as script and after_success are overridden in the doc stage, they will not be called.
I think you have to:
* remove `DOCS="0"` from the env
* remove the conditions here
* move the `pip install` for sphink in the `install` of the doc stage

File: .travis.yml [hunk 2]
```diff
@@ -41,8 +56,12 @@ install:
...
+services:
+  - postgresql
+
+
 script:
   - travis_run_tests
 
 after_success:
-  - travis_after_tests_success
+  - if [ "$DOCS" != "1" ] ; then travis_after_test_success; fi
```
Comment (guewen) [inline .travis.yml]:
```suggestion
  - travis_after_test_success
```

File: .travis.yml [hunk 3]
```diff
@@ -31,6 +32,20 @@ jobs:
...
+      install:
+        - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+        - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+        - travis_install_nightly
+        - pip install sphinx sphinx_bootstrap_theme sphinx-intl
+        - pip install git+https://github.com/OCA/odoo-sphinx-autodoc.git
+      script: ./.travis_build_doc
+      after_success: ./.travis_push_doc
+      env:
+        - DOCS="1"
```
Comment (guewen) [inline .travis.yml:46]: you probably don't need this `DOCS` variable anymore

File: .travis.yml [hunk 4]
```diff
@@ -41,8 +56,12 @@ install:
...
+services:
+  - postgresql
+
+
 script:
   - travis_run_tests
 
 after_success:
-  - travis_after_tests_success
+  - travis_after_test_success
```
Comment (guewen) [inline .travis.yml]: oops
```suggestion
  - travis_after_tests_success
```

File: .travis.yml (hunk 3 above)
Comment (guewen) [inline .travis.yml:46]:
```suggestion
```

**Comment by simahawk** [comment]
To be rebased on #356 as it gets merged.

**Comment by lmignon** [comment]
@damdam-s Do plan to fix the last [comment](https://github.com/OCA/connector/pull/355#issuecomment-542607053) from @simahawk ? This will allow this PR to be finalized and progress with the following ones (#357 ).

**Comment by damdam-s** [comment]
@lmignon I won't have the time this week. BTW I can add collaborators to push on my branch if someone need this quickly

**Comment by lmignon** [comment]
@damdam-s 

> @lmignon I won't have the time this week. BTW I can add collaborators to push on my branch if someone need this quickly

Yes do it. Good idea

**Comment by damdam-s** [comment]
@lmignon Should I add "just" you or do you suggest other people? C2C people can automatically push on my branch

**Comment by lmignon** [comment]
@damdam-s @guewen @simahawk I have finalized this work. However, I can't understand why runbot is in red...

**Comment by simahawk** [comment]
I cannot access logs :(
In any case it seems just a warning, is yellow.

**Comment by lmignon** [comment]
@damdam-s @guewen @simahawk Can we move forward with this one? If you want, I can squash the commits with the same descrition. I would like to get 'connector' available in 13 as soon as possible as these addons are the base of shopinvader...

**Review by simahawk — APPROVED** [review]
@lmignon I don't have time now for a detailed review. What I've seen so far looks good and there's no better way to test if this all works than running it on a real project (shopinvader in this case).

EDIT: Guewen is on holidays, I asked @sebalix to pass by for a quick review

**Comment by simahawk** [comment]
replaced by https://github.com/OCA/connector/pull/358

**Pull request closed without merging**